app/models/tslr_claim.rb

```diff
@@ -65,4 +71,16 @@ def ineligible_claim_school?
   def update_current_school
     self.current_school = employed_at_claim_school? ? claim_school : nil
   end
+
+  def normalise_trn
+    self.teacher_reference_number = normalised_trn
+  end
+
+  def normalised_trn
+    teacher_reference_number.gsub(/\D/, "")
+  end
+
+  def normalised_trn_length
```

Comment:
I think this method could better explain what it is doing. At the moment it kinda reads like it returns the length of a normalised TRN, and also the "normalised" bit feels more like a side effect of the implementation, rather than the behaviour we are trying to capture. How about something like `trn_must_be_seven_digits` instead?